faasmcli/faasmcli/tasks/docker_tasks.py

```diff
@@ -4,45 +4,30 @@
 
 from copy import copy
 from os.path import join, exists
-from subprocess import call, check_output
+from subprocess import run, PIPE
 
-import jinja2
 from invoke import task
 
 from faasmcli.util.env import PROJ_ROOT
 from faasmcli.util.version import get_faasm_version
 
-# Order matters here
```

Comment:
These Docker tasks are a lot simpler now because a) the actual build/ release part is done in Github; b) we no longer have container-specific dockerignore files